# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../fbmodes/strtod.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./xf86mode.cc.orig	2010-04-29 07:23:23.000000000 -0400
+++ ./xf86mode.cc	2010-04-29 07:23:45.000000000 -0400
@@ -3,6 +3,7 @@
 #include <vector>
 #include <cstring>
 #include <unistd.h>
+#include <cstdlib>
 
 /*
     This program has the purpose of explaining an X11 modeline.
--- ./argh/argh.cc.orig	2010-04-29 07:34:30.000000000 -0400
+++ ./argh/argh.cc	2010-04-29 07:36:22.000000000 -0400
@@ -1,6 +1,7 @@
 #include <map>
 #include <cstdio>
 #include <cstring>
+#include <cstdlib>
 
 using std::strchr;
 using std::sprintf;
